name: shacl validation of abox

on:
  push:
    branches:
    - develop-3.0.0

jobs:
  testexit:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
      with:
        ref: develop-3.0.0
        path: main
    - name: call 0
      continue-on-error: true
      run: |
           trap 'bash main/.github/workflow_resources/pyshacl_exitcode.sh $?' EXIT
           exit 0
    - name: call 1
      continue-on-error: true
      shell: bash {0}
      run: |
           trap 'bash main/.github/workflow_resources/pyshacl_exitcode.sh $?' EXIT
           exit 1
    - name: call 2
      continue-on-error: true
      run: |
           trap 'bash main/.github/workflow_resources/pyshacl_exitcode.sh $?' EXIT
           exit 2
    - name: call 3
      continue-on-error: true
      run: |
           trap 'bash main/.github/workflow_resources/pyshacl_exitcode.sh $?' EXIT
           exit 3
    - name: call 4
      continue-on-error: true
      run: |
           trap 'bash main/.github/workflow_resources/pyshacl_exitcode.sh $?' EXIT
           exit 4